examples/.gitignore

```diff
@@ -0,0 +1,4 @@
+!factors/Cargo.lock
```

Comment:
This entry doesn't make sense to me, because with a workspace, we only have a single `Cargo.lock` file for the whole workspace.

Reply:
Actually, I think the contents of this file could maybe be:
```
!Cargo.lock
```

Which should recursively allow any `Cargo.lock` file

Reply:
Thanks for the catch! I did fixes in the wrong order and confused myself about which lock files would actually exist if I was cleaning up correctly :)